convex/engine/schema.ts

```diff
@@ -1,7 +1,7 @@
 import { defineTable } from 'convex/server';
-import { v } from 'convex/values';
+import { Infer, v } from 'convex/values';
 
-const inputs = v.object({
+const input = v.object({
```

Comment:
I like this pattern: the fields are singular, which allows you to even export the table as `inputs`. This would allow listing all the tables in the top-level schema, so I don't have to navigate around to see the shape of data and have to think "would agents be in aiTown schema, agentSchemas" etc.